packages/svelte/src/compiler/phases/1-parse/state/tag.js

```diff
@@ -391,7 +391,7 @@ function open(parser) {
 			parameters: function_expression.params,
 			body: create_fragment(),
 			metadata: {
-				can_hoist: false,
+				can_hoist: undefined,
```

Comment:
This doesn't feel right, it should always be a boolean or if there's another state it should be an enum.

Reply:
I can make it an enum but it doesn't change the sense: undefined it's basically "don't know yet" because if the first visitor that sets `can_hoist` is the `bind:group` one  we don't want to override it within the snippet. In the end it will always be set by the snippet visitor and the fact that it's a falsy value allow us to default it to false without having to discriminate further.